test/readrows.ts

```diff
@@ -295,14 +293,15 @@ describe('Bigtable/ReadRows', () => {
   });
 
   it('should silently resume after server or network error', done => {
-    // 1000 rows must be enough to reproduce issues with losing the data and to create backpressure
-    const keyFrom = 0;
-    const keyTo = 1000;
-    // the server will error after sending this chunk (not row)
-    const errorAfterChunkNo = 423;
-
     service.setService({
-      ReadRows: readRowsImpl(keyFrom, keyTo, errorAfterChunkNo) as any,
+      ReadRows: readRowsImpl({
+        keyFrom: STANDARD_KEY_FROM,
+        keyTo: STANDARD_KEY_TO,
+        valueSize: VALUE_SIZE,
+        chunkSize: CHUNK_SIZE,
+        chunksPerResponse: CHUNKS_PER_RESPONSE,
+        errorAfterChunkNo: 423,
```

Comment:
any reason why we're using 423 or is it just an arbitrary spot to error?

Reply:
This was just the value that Alex used for this test. It looks like it is just a value that is as random as possible to make the test realistic.

Reply:
Curious if we could use `Math.random()` to test it more realistically.

Reply:
I added an additional test to emit an error at a random position. I want to keep the old test in place too so that if it breaks then we have predictable code we can use to debug the issue.